app/redux/redux-actions/adminActions.js

```diff
@@ -274,12 +384,13 @@ export const reduxGetAllPolicies = () => (dispatch) => {
   return { type: "DO_NOTHING", payload: null };
 };
 
-export const reduxGetAllEvents = () => (dispatch) => {
+export const reduxGetAllEvents = (cb) => (dispatch) => {
```

Comment:
can you explain what the callback would be used for here?

Reply:
Is what I use to detect when the request has returned a response. So I can update some state somewhere (like set "loading" to false)

Reply:
I've changed the logic for this now anyway. Callbacks wont be needed anymore.